submissions/jeffersonMormah/css/contact.css

```diff
@@ -226,6 +230,7 @@ main {
     text-align: center;
 }
 .footer1, .footer2, .footer3 {
+    width: 44%;
```

Comment:
You are declaring this twice??? @Jeffbuchi

Reply:
Rectified